spec/features/availability_spec.rb

```diff
@@ -52,9 +52,35 @@
 
     it 'that is an online resource and has no holdings to display' do
       visit '/?utf8=✓&search_field=all_fields&q=D-humanos+Arruti%2C+Mariana'
-      expect(page).not_to have_selector('button[data-target="#availability-22091400"]')
+      expect(page).not_to have_selector 'button[data-target="#availability-22091400"]'
       expect(page).not_to have_selector '.availability-22091400'
     end
+
+    context 'when Hathi ETAS is enabled' do
+      before do
+        Settings.readonly_holds = false
```

Comment:
I think I'd like to see this setting name change from to something like `Settings.display_holdings`.

Reply:
I can change this but just to make it clear, not using this setting for this etas work.

Reply:
No I'm clear on that, just something that has been bugging me a bit, like always is hard to comprehend. Like what does it mean again? I always have to figure out again. Does it mean whether or not we display the button or the holds or? Just think it could be clearer.